mvn-defaults/pom.xml

```diff
@@ -36,9 +36,8 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
         <javadoc.name>nosqlbench</javadoc.name>
-        <jacoco.version>0.8.8</jacoco.version>
```

Comment:
to be added back once 21 is supported?

Reply:
I moved the version down directly where it is used. There is no point in creating indirection on all things when there is only one reference.